test/testdefs.jl

```diff
@@ -1,7 +1,7 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
 function runtests(name, isolate=true)
-    Base.Test.DISABLE_TESTSET_PRINTING[] = true
+    Base.Test.testset_print_enable(false)
```

Comment:
That won't necessarily be the case if some form of https://github.com/JuliaLang/julia/issues/19567 happens. It's a global-state trap, best to avoid leaving it hiding for anyone who wants to refactor this to get surprised by later.